### DISCUSSION TITLE
[ StockWhiz ] : An interactive platform for stock analysis

### DISCUSSION THREAD

**Discussion by savakarrohan** [sympy/sympy#26539] [Show and tell]
This post is no longer valid. 

Sorry for the inconvenience.

**Comment by oscarbenjamin** [comment]
This doesn't seem relevant to sympy.

**Reply by SanjiS86** [reply]
Great post, I made a simple **[stock analysis app using JavaScript](https://github.com/SanjiS86/stockMarketApp/wiki)**. Anyone can download it and extract stock consensus target price, EPS data and social sentiment. Maybe you will find it useful.

**Comment by oscarbenjamin** [comment]
I've reported this as spam.

**Reply by moorepants** [reply]
I did also. I reported both.
